playground/monorepo-vite-condition/__tests__/resolve.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, expect, test } from 'vitest'
```

Comment:
This playground tests main/exports `vite` usage. There's many different patterns.